docs/docs/installation.md

```diff
@@ -109,123 +109,13 @@ You can refer [to this diff](https://github.com/software-mansion-labs/reanimated
 
 ## iOS
 
-Steps here are adapted directly from [React Native's RNTester app](https://github.com/facebook/react-native/blob/master/RNTester/RNTester/AppDelegate.mm), that is configured to use Turbo Modules.
-Most of the changes aren't specific to Reanimated but rather to Turbo Modules itself.
-If your iOS app is already using Turbo Modules, you can likely skip some of the steps below.
-If not, after making those changes your app will be compatible with Turbo Modules which may help with future React Native upgrades.
-
-1. `cd ios && pod install && cd ..`
-
-2. Rename `AppDelegate.m` to `AppDelegate.mm`.
-   > **_NOTE:_** It's important to do it with Xcode.
-3. Add AppDelegate category in `AppDelegate.mm`.
-
-```objectivec {1-2,4-7}
-#import <React/RCTCxxBridgeDelegate.h>
-#import <ReactCommon/RCTTurboModuleManager.h>
-...
-@interface AppDelegate() <RCTCxxBridgeDelegate, RCTTurboModuleManagerDelegate> {
-    RCTTurboModuleManager *_turboModuleManager;
-}
-@end
-```
-
-4. Enable TurboModules in `AppDelegate.mm`.
+On iOS installation is automatic. 
 
-```objectivec {3}
-  + (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
-  {
-    RCTEnableTurboModule(YES); // <- add
-```
+> **_NOTE:_** In previous releases, the installation process was manual and required turning turbo modules on.  Because we had 
+> reported some issues regarding bugs in turbo versions of react-native libraries we decided to changed our approach and we no longer 
+> use the standard way for registering a turbo module. It let us simplify the installation process and as a result, you can safely 
+> undo all installation steps from the previous instruction [previous instruction](https://docs.swmansion.com/react-native-reanimated/docs/installation#ios).  
```

Comment:
```suggestion
> undo all installation steps from the [previous instruction](https://docs.swmansion.com/react-native-reanimated/docs/installation#ios).  
```

Reply:
apparently the last stable version is not available under /docs/2.0.0-alpha.7 link – so the link here will no longer point to the correct instruction when we publish next version